jest.setup.js

```diff
@@ -1,4 +1,9 @@
 const { chrome } = require('jest-chrome/lib/index.cjs');
+const { SVGPathElement } = require('svgdom');
+
```

Comment:
I think you need to define `REACT_APP_VERSION` here

Reply:
yes, done